tests/ballerina-compiler-api-test/src/test/java/io/ballerina/semantic/api/test/SymbolBIRTest.java

```diff
@@ -433,4 +438,30 @@ public void testIntersectionTypeDefSymbolPosBIR() {
         assertEquals(lineRange.endLine().line(), 147);
         assertEquals(lineRange.endLine().offset(), 2);
     }
+
+    @Test
+    public void testObjectTypeSymbolDefQualifiers() {
+        Project project = BCompileUtil.loadProject("test-src/object_symbol_qualifiers.bal");
+        Package currentPackage = project.currentPackage();
+        Document srcFile = getDocumentForSingleSource(project);
+        SemanticModel model = getDefaultModulesSemanticModel(project);
+        
+        Optional<Symbol> symbol = model.symbol(srcFile, from(19, 20));
+        assertTrue(symbol.isPresent());
+        assertEquals(symbol.get().kind(), TYPE);
+        assertEquals(((TypeReferenceTypeSymbol) symbol.get()).typeDescriptor().typeKind(), OBJECT);
```

Comment:
Move to separate variables when possible so that you can reuse

Reply:
Fixed in [#2662](https://github.com/ballerina-platform/ballerina-lang/pull/40901/commits/26627582f1e188891880ffecb62c539c855b662f)